build.gradle

```diff
@@ -6,11 +6,23 @@ buildscript {
     }
     dependencies {
         classpath "com.gradle.publish:plugin-publish-plugin:0.11.0"
+        classpath "com.vanniktech:gradle-maven-publish-plugin:0.14.2"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.4.30"
     }
 }
 
 apply plugin: "com.gradle.plugin-publish"
 apply plugin: 'groovy'
+apply plugin: "com.vanniktech.maven.publish"
```

Comment:
and doubt plugin needs this

Reply:
AH yep sorry was using this for testing, will remove